src/test/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolverTest.scala

```diff
@@ -39,29 +42,6 @@ class InstanceUpdateOpResolverTest extends UnitTest with Inside {
       verifyNoMoreInteractions()
     }
 
-    "LaunchOnReservation for an unknown task" in new Fixture {
```

Comment:
I don't understand why this test was removed without replacement.

Reply:
cc @alenkacz

Reply:
LaunchOnReservation was removed in this diff, that does not exist anymore